README.md

```diff
@@ -474,4 +474,4 @@ Note: This project is unrelated to [hughperkins/pytorch](https://github.com/hugh
 
 ## License
 
-PyTorch has a BSD-style license, as found in the [LICENSE](LICENSE) file.
```

Comment:
You could try: git checkout main -- README.md to override this file with main's README.md